README.md

```diff
@@ -28,6 +33,10 @@ docker compose up
 
 That's it! If you have questions, please reach out to us.
 
+### Cloud deployment
```

Comment:
I'm not sure we will provide this. We'll have a github action at some point that does it, but I don't know that we'll have any documentation around it.

Reply:
Removed...